src/components/NumberInput/NumberInput.md

```diff
@@ -48,3 +48,58 @@ const dollarMaskFunction = createNumberMask({
   }}
 />
 ```
+
+This one sets a `solid lock icon`, which results in the number input with solid lock icon.
+```jsx
+import { NumberInput } from './index.js';
+// formChangeHandler gets wired up automatically if using <Form /> component
+const formChangeHandlerStub = () => {}
+;<NumberInput
+  initialValue="123"
+  name="the-number-input-example"
+  allCaps={true}
+  labelCopy="Enter a number (must be even to validate)"
+  data-tid='the-number-input'
+  placeholder='number input'
+  formChangeHandler={formChangeHandlerStub}
+  validator={(n) => {
+    if (n > Number.MAX_SAFE_INTEGER) {
+      return 'Number too large—you have exceeded JavaScript\s powers!!'
+    }
+    return n % 2 === 0 ? '' : 'Must be an even number'
+  }}
+  iconPrefix="fas"
```

Comment:
For example, as a user I wouldn't know what `fas` and `far` are and when/how to use them for the icon